doc.md

```diff
@@ -0,0 +1,23 @@
+# Project Scalability
+- for this project future we should add an infinite scroll system that does the pagination as you scroll
```

Comment:
Do you have any proposals? How would you go about the integration of such system?